notebooks/215-image-inpainting/215-image-inpainting.ipynb

```diff
@@ -77,19 +81,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "precision = \"FP32\"\n",
-    "ir_path = Path(f\"{base_model_dir}/public/{model_name}/{precision}/{model_name}.xml\")\n",
+    "model_dir = Path(base_model_dir, 'public', 'ir')\n",
+    "ir_path = Path(f\"{model_dir}/frozen_model.xml\")\n",
     "\n",
-    "# Run Model Optimizer if the OpenVINO IR model file does not exist.\n",
+    "# Run Model Optimizer to convert model to OpenVINO IR FP32 format, if the IR file does not exist.\n",
     "if not ir_path.exists():\n",
-    "    print(\"Exporting TensorFlow model to IR... This may take a few minutes.\")\n",
-    "    convert_command = f\"omz_converter \" \\\n",
-    "                      f\"--name {model_name} \" \\\n",
-    "                      f\"--download_dir {base_model_dir} \" \\\n",
-    "                      f\"--precisions {precision}\"\n",
-    "    ! $convert_command\n",
+    "    mo_command = f'mo --input_model {model_path} --input Placeholder,Placeholder_1 --input_shape [1,512,680,3],[1,512,680,1] --output=Minimum --output_dir {model_dir}'\n",
```

Comment:
Can we try to convert this model without specification --input, --output and --input_shape?

Reply:
I tried and on step:
`N, H, W, C = input_layer.shape`
got error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-02fb957c3494> in <module>
----> 1 N, H, W, C = input_layer.shape

ValueError: get_shape was called on a descriptor::Tensor with dynamic shape
```

Reply:
pk, then let's stay as is for now